govc/vcsa/shutdown/poweroff.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright (c) 2021 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package shutdown
+
+import (
+	"context"
+	"flag"
+	"fmt"
+
+	"github.com/vmware/govmomi/govc/cli"
+	"github.com/vmware/govmomi/govc/flags"
+	"github.com/vmware/govmomi/vapi/appliance/shutdown"
+)
+
+type powerOff struct {
+	*flags.ClientFlag
+	*flags.OutputFlag
+
+	reason string
+	delay  int
```

Comment:
question: is this seconds? Not clear from description. Wondering if we can/should use time.Duration? (can you check what other govc commands are using?)

Reply:
It would be great if you could simply drop a comment behind the line indicating it's minutes right there in the code, e.g.:

```suggestion
	delay  int // in minutes
```

Reply:
updated `delay  int // in minutes` in reboot.go, poweroff.go files